src/feature/control/control_getinfo.c

```diff
@@ -1660,6 +1692,16 @@ static const getinfo_item_t getinfo_items[] = {
        "Onion services detached from the control connection."),
   ITEM("sr/current", sr, "Get current shared random value."),
   ITEM("sr/previous", sr, "Get previous shared random value."),
+  PREFIX("rephist/ntor", rephist, "NTor circuit handshake stats."),
+  ITEM("rephist/ntor/onion_handshakes_assigned", rephist,
+       "Assigned NTor circuit handshake stats."),
+  ITEM("rephist/ntor/onion_handshakes_requested", rephist,
+       "Requested NTor circuit handshake stats."),
+  PREFIX("rephist/tap", rephist, "TAP circuit handshake stats."),
+  ITEM("rephist/tap/onion_handshakes_assigned", rephist,
+       "Assigned TAP circuit handshake stats."),
+  ITEM("rephist/tap/onion_handshakes_requested", rephist,
+       "Requested TAP circuit handshake stats."),
```

Comment:
See torspec comment about the naming. We'll wait on a decision there.

Reply:
So, the `PREFIX` are not correct.

But also I would propose we use `stats/ntor/requested` and `stats/ntor/assigned`.

Reason I say that is that I think `onion` and `handshake` are a bit redundant since `ntor` is by definition an onion handshake.